provider/infoblox/infoblox.go

```diff
@@ -530,8 +609,14 @@ func (p *ProviderConfig) recordSet(ep *endpoint.Endpoint, getObject bool, target
 		obj.Name = ep.DNSName
 		obj.Ipv4Addr = ep.Targets[targetIndex]
 		obj.View = p.view
```

Comment:
Do we still need them taking into account the query params we pass now?

Reply:
Most likely, I forgot to remove this.

Reply:
No, right now that is because it is needed on the lines with recordSet := infobloxRecordSet{ ... }